phasespace/fromdecay/__init__.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+from .genmultidecay import GenMultiDecay  # noqa: F401
+
+try:
+    import zfit  # noqa: F401
+    import zfit_physics as zphys  # noqa: F401
+    from particle import Particle  # noqa: F401
+except ModuleNotFoundError as error:
+    raise ModuleNotFoundError(
+        "The fromdecay functionality in phasespace requires particle and zfit-physics. "
+        "Either install phasespace[fromdecay] or particle and zfit-physics.",
+        file=sys.stderr,
+    ) from error

```

Comment:
Could probably add in this file the following:

```
__all__ = ("GenMultiDecay")

def __dir__():
    # type: () -> Tuple[str, ...]
    return __all__
```

Though CI checks could complain (depending on what hooks you have, did not check) that `__all__` does not contain all that you import. You could delete the imported modules if that's necessary.

Reply:
Added in 0b6ec078e05213984811275d063028b07a477c1d